dbms/src/Core/Settings.h

```diff
@@ -102,6 +102,10 @@ struct Settings : public SettingsCollection<Settings>
     M(SettingBool, distributed_group_by_no_merge, false, "Do not merge aggregation states from different servers for distributed query processing - in case it is for certain that there are different keys on different shards.") \
     M(SettingBool, optimize_skip_unused_shards, false, "Assumes that data is distributed by sharding_key. Optimization to skip unused shards if SELECT query filters by sharding_key.") \
     \
+    M(SettingBool, input_format_parallel_parsing, true, "Enable parallel parsing for several data formats (JSON, TSV, TKSV, Values, CSV).") \
+    M(SettingUInt64, max_threads_for_parallel_reading, 0, "The maximum number of threads to parallel reading. By default, it is set to max_threads.") \
+    M(SettingUInt64, min_chunk_size_for_parallel_reading, 4, "The minimum chunk size, which each thread tries to parse under mutex in parallel reading.") \
```

Comment:
Just curious, what this "4"  means? Looks like number of chunks, but looking at CSVRowInputStream.cpp `while` loop it looks like 4 bytes. Am I missing something?